tensilelite/Tensile/Tests/common/gemm/dtv.yaml

```diff
@@ -0,0 +1,311 @@
+        - ScheduleIterAlg: [3]
+        - ExpandPointerSwap: [0,1]
+        - LdsBlockSizePerPadA: [-1]
+        - LdsBlockSizePerPadB: [-1]
+        - StorePriorityOpt: [0]
+        - VectorStore: [-1]
+        - StoreSyncOpt: [0]
+        - LdsPadA: [-1]
+        - LdsPadB: [-1]
+        - 1LDSBuffer: [1]
+        - GlobalSplitU: [1,2]
+        - GlobalSplitUAlgorithm: ["MultipleBuffer", "MultipleBufferSingleKernel"]
+        - SourceSwap: [0, 1]
+        - DirectToVgprA: [0,1]
+      BenchmarkJoinParameters:
+      BenchmarkFinalParameters:
+        - ProblemSizes:
+          - Exact: [255,   255, 1,  15]
+          - Exact: [255,   255, 1,  17]
+          - Exact: [255,   255, 1,  32]
+          - Exact: [255,   255, 1,  33]
+          - Exact: [255,   255, 1,  47]
+          - Exact: [255,   255, 1,  63]
+          - Exact: [255,   255, 1,  79]
+          - Exact: [255,   255, 1,  95]
+          - Exact: [255,   255, 1, 111]
+          - Exact: [255,   255, 1, 127]
+          - Exact: [255,   255, 1, 191]
+          - Exact: [255,   255, 1, 256]
+
+  ########################################
+  # HHS TN DTVA + BIAS + Activation
+  ########################################
+  -
+    - # ProblemType
+      OperationType: GEMM
+      DataType: h
+      DestDataType: h
+      ComputeDataType: s
+      HighPrecisionAccumulate: True
+      TransposeA: 0
+      TransposeB: 0
+      UseBeta: True
+      Batched: True
+      UseBias: 1
+      Activation: True
+      UseScaleAlphaVec: 1
+      UseScaleAB: "Scalar"
+    - # BenchmarkProblemSizeGroup - Standard - All problem
+      InitialSolutionParameters:
+      BenchmarkCommonParameters:
+        - KernelLanguage: ["Assembly"]
+      ForkParameters:
+        - MatrixInstruction:
+          - [16, 16, 16, 1,  1,   2, 1,  4,1 ]
+          - [16, 16, 16, 1,  1,   4, 1,  4,1 ]
+          - [32, 32,  8, 1,  1,   2, 1,  4,1 ]
+          - [32, 32,  8, 1,  1,   4, 1,  4,1 ]
+        - WorkGroup:
+          - [16,16,1]
+        - GlobalReadVectorWidthA: [2,4]
+        - GlobalReadVectorWidthB: [2,4]
+        - PrefetchGlobalRead: [1,2]
+        - PrefetchLocalRead: [1]
+        - ClusterLocalRead: [1]
+        - NumElementsPerBatchStore: [0]
+        - DepthU: [128]
+        - VectorWidthA: [-1]
+        - VectorWidthB: [-1]
+        - MIArchVgpr: [0, 1]
+        - LocalWritePerMfma: [-1]
+        - StaggerU: [4]
+        - StaggerUStride: [256]
+        - StaggerUMapping: [2]
+        - WorkGroupMapping: [1]
+        - ScheduleIterAlg: [3]
+        - ExpandPointerSwap: [0]
+        - TransposeLDS: [1]
+        - LdsBlockSizePerPadA: [-1]
+        - LdsBlockSizePerPadB: [-1]
+        - StorePriorityOpt: [0]
+        - VectorStore: [-1]
+        - StoreSyncOpt: [0]
+        - LdsPadA: [-1]
+        - LdsPadB: [-1]
+        - 1LDSBuffer: [1]
+        - GlobalSplitU: [1,2]
+        - GlobalSplitUAlgorithm: ["MultipleBuffer", "MultipleBufferSingleKernel"]
+        - SourceSwap: [0, 1]
```

Comment:
DTVA is not enabled.

Reply:
Thanks for pointing this out.
I added DTVA=1.